keras/layers/convolutional.py

```diff
@@ -343,6 +343,194 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
+class Convolution3D(Layer):
+    '''Convolution operator for filtering windows of three-dimensional inputs.
+    When using this layer as the first layer in a model,
+    provide the keyword argument `input_shape`
+    (tuple of integers, does not include the sample axis),
+    e.g. `input_shape=(3, 10, 128, 128)` for 10 frames of 128x128 RGB pictures.
+
+    Note: this layer will only work with Theano for the time being.
+
+    # Input shape
+        5D tensor with shape:
+        `(samples, channels, len_conv_dim1, len_conv_dim2, len_conv_dim3)` if dim_ordering='th'
+        or 5D tensor with shape:
+        `(samples, len_conv_dim1, len_conv_dim2, len_conv_dim3, channels)` if dim_ordering='tf'.
+    # Output shape
+        5D tensor with shape:
+        `(samples, nb_filter, len_new_dim1, len_new_dim2, len_new_dim3)` if dim_ordering='th'
+        or 5D tensor with shape:
+        `(samples, len_new_dim1, len_new_dim2, len_new_dim3, nb_filter)` if dim_ordering='tf'.
+        `len_new_dim1`, `len_new_dim2` and `len_new_dim3` values might have changed due to padding.
+    # Arguments
+        nb_filter: Number of convolution filters to use.
+        len_conv_dim1: Length of the first dimension in the covolution kernel.
```

Comment:
It is a bit confusing that `len_conv_dim1` is defined here as a property of the convolution kernel, but is also used above when defining the input shape. It should be easy to distinguish if a shape is for the image or kernel.